etsin_finder/frontend/js/components/dataset/sidebar/index.jsx

```diff
@@ -141,7 +143,7 @@ class Sidebar extends Component {
               <SidebarItem>
                 <Logo
                   lang={getDataLang(catalogTitle)}
-                  alt={catalogPublisherHomepage}
+                  alt={translate('dataset.catalog_alt_text', { title })}
```

Comment:
Doesn't trigger a reactive update when language changes, which doesn't matter much here since other parts of the component do. Also noticed that the `Logo` component doesn't actually use the `lang` prop. Anyways, this would be more reliably reactive:
```
                <Translate
                  component={Logo}
                  attributes={{ alt: 'dataset.catalog_alt_text' }}
                  with={{ title }}
                  file={logo}
                  url={catalogPublisherHomepage}
                />
```

Hmm. Seems that the `Image` component in `general/image.jsx` used by `Logo` actually prevents reactive updates by storing the props in state in the constructor. I'm pretty sure that's pointless, and the component could be simplified into this: 
```
const Image = ({ alt, file }) => <StyledImg alt={alt} src={file} />
```